### ISSUE TITLE
Fix contrast ratio of search placeholder text

### ISSUE THREAD

**Issue by brampitoyo** [mozilla-mobile/focus-android#3366]
<img width="756" alt="screen shot 2018-09-10 at 5 01 13 pm" src="https://user-images.githubusercontent.com/51007/45277360-7535ba80-b51b-11e8-9d06-acdf8b5bce27.png">

Feedback from TPG:

> The search placeholder text (`#C79CAF`) fails the contrast minimum of 4.5:1 against the pink background (`#7B144C`) at 4.3:1.
> 
> This should exceed 4.5:1.

Here again, upping the text opacity from `grey-10` 60% to `grey-10` 72% will increase the contrast to 5.6:1, making it pass WCAG Double A.

#### Current: `grey-10` 60%

<img width="360" src="https://user-images.githubusercontent.com/51007/45277442-e83f3100-b51b-11e8-8945-d1e182e0f416.png">

#### Proposed: `grey-10` 72%

<img width="360" src="https://user-images.githubusercontent.com/51007/45277478-20467400-b51c-11e8-9b01-7ff3768d1abd.png">
